recipes-debian/doxygen/doxygen/0001-build-don-t-look-for-Iconv.patch

```diff
@@ -0,0 +1,13 @@
+--- a/CMakeLists.txt.origi	2021-10-13 10:17:14.201162558 +0900
++++ b/CMakeLists.txt	2021-10-13 10:21:16.008444078 +0900
+@@ -73,10 +73,6 @@ if (sqlite3)
+     find_package(SQLite3 REQUIRED)
+ endif()
+ 
+-find_package(Iconv REQUIRED)
+-include_directories(${ICONV_INCLUDE_DIR})
+-
+-
+ #set(DOXYDOCS ${CMAKE_SOURCE_DIR}/doc CACHE INTERNAL "Path to doxygen docs")
+ set(DOXYDOCS ${PROJECT_BINARY_DIR}/doc)
+ set(ENV{DOXYGEN_DOCDIR} ${DOXYDOCS})
```

Comment:
Please use "git format-patch" to create a patch.
I created it this time for you.
https://gist.github.com/hiraku-wfs/80506f70fd06d54d3fad9f80c9ee0d38

You can find corresponding git repository from Debian's "Developer Information".

1. https://packages.debian.org/source/buster/doxygen
2. https://tracker.debian.org/pkg/doxygen
3. https://salsa.debian.org/debian/doxygen/-/tree/debian/buster <--